cxx/distributions/stringcat_test.cc

```diff
@@ -0,0 +1,24 @@
+// Apache License, Version 2.0, refer to LICENSE.txt
+
+#define BOOST_TEST_MODULE test StringCat
+
+#include "distributions/stringcat.hh"
+
+#include <boost/test/included/unit_test.hpp>
+namespace tt = boost::test_tools;
+
+BOOST_AUTO_TEST_CASE(test_simple) {
```

Comment:
Could you call `sample` in the test somewhere?

Reply:
Done.